internal/controller/apiextensions/claim/reconciler_test.go

```diff
@@ -825,7 +825,13 @@ func TestReconcile(t *testing.T) {
 
 			got, err := r.Reconcile(context.Background(), reconcile.Request{})
 
-			if diff := cmp.Diff(tc.want.claim, tc.args.claim, cmpopts.EquateApproxTime(30*time.Second)); diff != "" {
+			if diff := cmp.Diff(tc.want.claim, tc.args.claim, cmpopts.AcyclicTransformer("StringToTime", func(s string) any {
```

Comment:
Might be useful to add this transformer to `crossplane-runtime` in the future!

Reply:
👍 if we agree on a final form here, I can freeze this PR, add it to crossplane-runtime, release a new version and switch this to consume that instead.

Reply:
@phisco I don't think this needs to be blocker for fixing up these tests 👍🏻